features/testbot/trip.feature

```diff
@@ -85,13 +175,106 @@ Feature: Basic trip planning
             | pq    |
             | qm    |
 
+        When I plan a trip I should get
+            | waypoints                       | status         | message                                      |
+            | a,b,c,d,e,f,g,h,i,j,k,l,m,n,o,p | NoTrips        | No trip visiting all destinations possible.  |
+
+# Test TFSE
+    Scenario: Testbot - Trip: TFSE with errors
+        Given the node map
+            """
+            a b
+
+            d c
+            """
+
+         And the ways
+            | nodes |
+            | ab    |
+            | dc    |
+
+         When I plan a trip I should get
+            |  waypoints    | source  | destination | roundtrip | status           | message                                       |
+            |  a,b,c,d      | first   | last        | false     | NoTrips          | No trip visiting all destinations possible.   |
```

Comment:
Same here: `source` and `destination` should be query parameters not part of the result table.